src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs

```diff
@@ -21,13 +21,16 @@ namespace Microsoft.NET.HostModel.Bundle
     public class TargetInfo
     {
         public readonly OSPlatform OS;
+        public readonly Architecture Arch;
         public readonly Version FrameworkVersion;
         public readonly uint BundleVersion;
         public readonly BundleOptions DefaultOptions;
+        public readonly int AssemblyAlignment;
```

Comment:
Do we need to store the AssemblyAlignment in a field? It looks like it could be a calculated property based on the Arch. Moreover, do we need to store it on `TargetInfo` at all? It's not really a property of the target info. It was previously stored in Bundler.cs, which seems like the only place it's needed.

Reply:
I made it a field for consistency with `DefaultOptions` and `BundleVersion`, but don't have a strong preference. To me it seems like it belongs with `TargetInfo` because it depends on the alignment requirements of the target platform - for example if the bundler accepted an option to change the alignment, we would still only allow multiples of this value (similar to `DefaultOptions`).

Reply:
> belongs with TargetInfo because it depends on the alignment requirements of the target platform

I would disagree -- `TargetInfo` is a collection of data about the target, while alignment is a piece of data about the bundler, which is dependent on `TargetInfo`. My general strategy would be for TargetInfo to carry only the essentials of the target, because that makes it more reusable and lighter weight for dependent components.

However I think this is all pretty academic because all of this code is pretty small, so it doesn't particularly matter to me either way 😄

Reply:
Point taken - I'll leave it this way for consistency, but we could consider removing bundler-related decisions from it.